lib/chef/knife/core/windows_bootstrap_context.rb

```diff
@@ -49,8 +49,12 @@ def validation_key
           end
         end
 
+        def encrypted_data_bag_secret
+          escape_and_echo(@secret)
+        end
+
         def secret
```

Comment:
Could we just make this an accessor?

Reply:
It's overloading `encrypted_data_bag_secret` in `bootstrap_context.rb`

Reply:
oh nevermind

Reply:
made it an accessor